cmd/manager/main.go

```diff
@@ -96,7 +100,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	duration := 30 * time.Second
+	duration := defaultSyncInterval
+	if str, ok := os.LookupEnv("LIEUTENANT_SYNC_DURATION"); ok {
+		if d, err := time.ParseDuration(str); err != nil {
+			log.Error(err, "failed to parse LIEUTENANT_SYNC_DURATION")
+		} else if d > 30*time.Second {
+			duration = d
+		} else {
+			log.Info("LIEUTENANT_SYNC_DURATION is less than 30s, setting to 30s")
```

Comment:
The default is not 30s
```suggestion
			log.Infof("LIEUTENANT_SYNC_DURATION is less than 30s, setting to %s", defaultSyncInterval)
```

Reply:
This is not meant to enforce the default as a minimal value, but rather to enforce more than 30 seconds. We saw that even 30 seconds is not enough, so setting it lower than that doesn't make sense.

Also the `logr` interface does not support `Infof()`.

Reply:
Fine by me, but then you need to do `duration = 30 * time.Second`